src/format/opamFile.ml

```diff
@@ -823,10 +826,13 @@ module ConfigSyntax = struct
                (kind,criterion)::List.remove_assoc kind t.solver_criteria }
   let with_solver t solver = { t with solver = Some solver }
 
+  let backup_url_ocamlpro = "http://opam.ocamlpro.com/backup"
```

Comment:
On Tue, Mar 01, 2016 at 12:38:27AM -0800, Daniel Bünzli wrote:
> > @@ -823,10 +826,13 @@ module ConfigSyntax = struct
> >                 (kind,criterion)::List.remove_assoc kind t.solver_criteria }
> >    let with_solver t solver = { t with solver = Some solver }
> >  
> > +  let backup_url_ocamlpro = "http://opam.ocamlpro.com/backup"
> 
> > but what's wrong in OCamlPro offering this service and some sysadmin time ?
> 
> OCamlPro's website has been 404ing more than once with stuff hosted there (the last occurence being that [one](https://github.com/ocaml/opam/issues/2438)). We don't want to wait for an OCamlPro employee to show up to be able to fix these problems.

Sure, mistakes were made during a server migration and things were quickly fixed.